Sources/SkipFirebaseMessaging/SkipFirebaseMessaging.swift

```diff
@@ -192,7 +192,41 @@ public class MessagingService : FirebaseMessagingService {
         } else {
             attachments = []
         }
-        let content = UNNotificationContent(title: notification.title ?? "", body: notification.body ?? "", userInfo: userInfo, attachments: attachments)
+        var title = notification.title
+        if title == nil, let titleKey = notification.getTitleLocalizationKey() {
+            let localized = Bundle.main.localizedString(forKey: titleKey, value: nil, table: nil)
+            if let args = notification.getTitleLocalizationArgs() {
+                do {
+                    title = String(format: localized, arguments: skip.lib.Array(args.toList()))
+                } catch {
+                    android.util.Log.e("SkipFirebaseMessaging", "onMessageReceived, notification title (\(localized) format error", error as? Throwable )
+                    title = localized
+                }
+            } else {
+                title = localized
+            }
+        } else {
+            title = ""
```

Comment:
Won't this blank out the title if it was set in the notification and there was no localization key? It seems like this else clause can just be removed.